.github/workflows/CI.yml

```
@@ -124,6 +144,21 @@ jobs:
             ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="docker-cluster"
           fi
 
+      - name: Mixed cluster backwards compatibility tests
+        run: |
+          echo "Running mixed cluster backwards compatibility tests ..."
+          ./gradlew adBwcCluster#mixedClusterTask -DmixedCluster=123 -Dtests.security.manager=false
```

I understand we are using mixedCluster variable to run some specific tests. 
Can we use a boolean flag instead of an arbitrary number like 123?
```suggestion
          ./gradlew adBwcCluster#mixedClusterTask -DmixedCluster=true -Dtests.security.manager=false
```

---

Makes sense, updated it to `true` for the flag.